DeepFried2/layers/SpatialConvolution3D.py

```diff
@@ -25,17 +25,23 @@ def __init__(self, n_input_vol, n_output_vol, k_w, k_h, k_d, with_bias=True, ini
 
     def symb_forward(self, symb_input):
 
-        """symb_input shape: (n_input, depth, channels, height, width)"""
+        """symb_input shape: (n_input, channels, depth, height, width)"""
 
         if symb_input.ndim < 5:
             raise NotImplementedError('3D convolution requires a dimension >= 5')
 
+        # shuffle bcd01 -> bdc01
+        symb_input = symb_input.dimshuffle(0,2,1,3,4)
+
         conv_output = conv3d2d.conv3d(symb_input,
                                       self.weight,
                                       filters_shape=self.w_shape,
                                       border_mode=self.border_mode)
 
+        # shuffle bdc01 -> bcd01
+        conv_output = conv_output.dimshuffle(0,2,1,3,4)
```

Comment:
Maybe `foo.swapaxes(1,2)` would be clearer here and above? But I'm also fine with keeping this as-is, I'll let you decide.